packages/gluestick-cli/new.js

```diff
@@ -59,16 +60,24 @@ module.exports = (appName, options, exitWithError) => {
   );
 
   spawn.sync(
-    options.yarn ? 'yarn' : 'npm',
+    !options.npm && which('yarn') !== null ? 'yarn' : 'npm',
     ['install'],
     {
       cwd: process.cwd(),
       stdio: 'inherit',
     },
   );
+  // Remove --npm or -n options cause this is no longer needed
+  const args = commander.rawArgs.slice(2);
+  if (args.indexOf('--npm') >= 0) {
+    args.splice(args.indexOf('--npm'), 1);
+  } else if (args.indexOf('-n') >= 0) {
+    args.splice(args.indexOf('-n'), 1);
+  }
```

Comment:
Yeah, we don't remove `--dev` or `-d` because we pass it to `gluestick` new command and there we use it. If we don't remove it we get error with message similar to `don't know option --npm/-n`. This is because we  remove option `--yarn/npm` from `gluestick` new command.